src/reducers/index.js

```diff
@@ -13,8 +13,16 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+    return {
+      ...state,
+      count: state.count +1
+    }
     case DECREMENT:
     // Fill in the body of this case
+    return {
+      ...state,
+      count: state.count -1
```

Comment:
great job making it easy to read by using the state count instead of the payload!